# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1253227141 14400
#      Thu Sep 17 18:39:01 2009 -0400
# Node ID d0e2aa1974e9680337ce0037f098bceb98ce2a70
# Parent  11844593875aef155cd078bd76147159d0458997
changelog fix for previous change

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,14 +1,14 @@
 2009-09-17  John W. Eaton  <jwe@octave.org>
 
 	* oct-types.h.in: Delete.
 	* Makefile.in (BUILT_INCLUDES, DISTFILES): Remove oct-types.h.in
 	from the list.
-	(distclean, maintainer-clean): 
+	(distclean, maintainer-clean): Don't remove oct-types.h
 	* DAEFunc.h, MatrixType.h, base-aepbal.h, dim-vector.h,
 	lo-specfun.h, lo-utils.h, mx-op-decl.h, oct-inttypes.h,
 	randgamma.h, randmtzig.h, randpoisson.h:
 	Don't include oct-types.h.
 
 2009-09-16  Jaroslav Hajek  <highegg@gmail.com>
 
 	* oct-rand.cc (octave_rand::do_matrix, do_nd_array, do_vector):
